tracing/src/main/java/io/airlift/tracing/OpenTelemetryModule.java

```diff
@@ -45,11 +51,12 @@ public OpenTelemetryModule(String serviceName, String serviceVersion)
     public void configure(Binder binder)
     {
         configBinder(binder).bindConfig(OpenTelemetryConfig.class);
+        newSetBinder(binder, SpanProcessor.class);
     }
 
     @Provides
     @Singleton
-    public OpenTelemetry createOpenTelemetry(NodeInfo nodeInfo, OpenTelemetryConfig config)
+    public OpenTelemetry createOpenTelemetry(NodeInfo nodeInfo, Set<SpanProcessor> extraSpanProcessors, OpenTelemetryConfig config)
```

Comment:
Maybe the span processor could be a dependency with a default, so it's possible to override it in tests?

Reply:
Yeah, makes sense.